IFC4x3/Properties/i/InjectionDistribution_2lnxV9oFn01AvbA6wexzaD/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="InjectionDistribution_2lnxV9oFn01AvbA6wexzaD" Name="InjectionDistribution" UniqueId="afc7b7c9-c8fc-4004-ae65-286ea8efd90d" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
How about renaming to "InjectionDistributionEquipment"?

Reply:
Agreed, sounds better.

Reply:
Name changed to InjectionDistributionEquipment.